tests/bot/cogs/test_jams.py

```diff
@@ -0,0 +1,145 @@
+import unittest
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from bot.cogs.jams import CodeJams, setup
+from bot.constants import Roles
+from tests.helpers import MockBot, MockContext, MockGuild, MockMember, MockRole, MockTextChannel
+
+
+class JamCreateTeamTests(unittest.IsolatedAsyncioTestCase):
+    """Tests for `createteam` command."""
+
+    def setUp(self):
+        self.bot = MockBot()
+        self.admin_role = MockRole(name="Admins", id=Roles.admins)
+        self.command_user = MockMember([self.admin_role])
+        self.guild = MockGuild([self.admin_role])
+        self.ctx = MockContext(bot=self.bot, author=self.command_user, guild=self.guild)
+        self.cog = CodeJams(self.bot)
+        self.utils_mock = patch("bot.cogs.jams.utils").start()
+        self.default_args = [self.cog, self.ctx, "foo"]
```

Comment:
Does this need to be attribute for all tests?
 It's only used in two places where it fits within the line limit (One call to createteam already does this) and the name doesn't particularly hint on what those default args are for